app/build.gradle

```diff
@@ -59,6 +59,13 @@ dependencies {
     def fragment_version = "1.3.6"
     implementation "androidx.fragment:fragment-ktx:$fragment_version"
 
+    // image loading library
+    implementation ("com.github.bumptech.glide:glide:4.11.0") {
+        exclude group: "com.android.support"
```

Comment:
we are using `androidx` which replaced `com.android.support` so we wouldn't need this. Posted glides GitHub below check out the dependencies there

https://github.com/bumptech/glide